client/a8c-for-agencies/sections/referrals/primary/commission-overview/index.tsx

```diff
@@ -99,7 +100,13 @@ export default function CommissionOverview( {
 						>
 							{ translate(
 								'You will receive a revenue share of 5 basis points (bps) on new WooPayments total payments volume (“TPV”) on client sites through June 30, 2025.' +
-									' For example, if your client’s store generates $1M in TPV per year, your revenue share for that year would be $500.'
+									" For example, if your client's store generates %(maxAmount)s in TPV per year, your revenue share for that year would be %(amount)s.",
+								{
+									args: {
+										maxAmount: formatCurrency( 1000000, 'USD' ),
```

Comment:
Probably we can stick to $1M here?

<img width="725" alt="Screenshot 2025-02-21 at 3 28 04 PM" src="https://github.com/user-attachments/assets/61ab3b5a-ecb7-4ea6-85e4-5b169a43d505" />

Reply:
Interesting! We should probably add a "compact" notation to `formatCurrency`. Let's stick with `$1M` and I can follow up on this separately. 

cc @sirbrillig / @Automattic/i18n  We can just allow `formatCurrency` to accept `numberFormatOptions` in the same way `numberFormat` does, so someone can just tailor the rendering a little better. Or otherwise we can create a `numberFormatCompactCurrency` abstraction. I'll create an issue.

Reply:
Thanks!

I think it's better to format this the same way. I did a workaround: https://github.com/Automattic/wp-calypso/pull/100135/commits/bd67c53b68c08057d034647a154f22f09957a2ac, a quick hack to get this right. Let me know if this works until we have a function for it. Or we could skip the formatting for this particular section for both $1M and $500.

<img width="775" alt="Screenshot 2025-02-21 at 5 14 06 PM" src="https://github.com/user-attachments/assets/442820ce-7158-4ef2-9ff1-eebefe415e51" />
<img width="775" alt="Screenshot 2025-02-21 at 5 18 51 PM" src="https://github.com/user-attachments/assets/11031fe0-4752-4217-a5b3-7c417d176e60" />

Reply:
This is perfect. Thanks! Good use of the currency object parts. Glad to see there is a workaround with the existing APIs :-)

Reply:
Awesome. I created a function so that we can easily replace all later: https://github.com/Automattic/wp-calypso/pull/100135/commits/fb2629d9de1e356ca02e886206c948f83e0e062b